ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -59,17 +59,26 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-ovn-ipsec-amd-mixarch-day2-f14
-  cron: 42 5 14,28 * *
+- as: aws-ipi-disc-priv-arm-mixarch-f28
```

Comment:
Similar comments with https://github.com/openshift/release/pull/54740#discussion_r1698220768

Reply:
Removed

Reply:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml#L121
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.17.yaml#L67


@jianlinliu do we also need to delete these two?

Reply:
They are y version, not include this pr

Reply:
> do we also need to delete these two

Yes, I think so.